content/knowledge-base/load-firebase-configuration.md

```diff
@@ -6,7 +6,7 @@ aliases:
   - '../custom-scripts/load-firebase-configuration'
 ---
 
-Instead of committing the Firebase configuration files to your repository, you can upload them to Codemagic as [environment variables](../building/environment-variables/) and reference them in a custom script. Note that the Firebase configuration file (`google-services.json` for Android or `GoogleService-Info.plist` for iOS) must be Base64-encoded.
+Instead of committing the Firebase configuration files to your repository, you can upload them to Codemagic as [environment variables](../building/environment-variables/) and reference them in a custom script. Note that the Firebase configuration file (`google-services.json` for Android or `GoogleService-Info.plist` for iOS) must be selected as **secure**.
 
 1.  Save your Firebase config files as environment variables, e.g. `ANDROID_FIREBASE_SECRET` and `IOS_FIREBASE_SECRET`. 
```

Comment:
Here I would mention the need to copy-paste the contents of the file to save them to env vars and clicking Secure to encrypt them.